app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlInteractor.kt

```diff
@@ -322,6 +330,21 @@ class SessionControlInteractor(
         controller.handleReadPrivacyNoticeClicked()
     }
 
+    override fun showWallpapersOnboardingDialog(state: AppState?): Boolean {
```

Comment:
I think `state` could be non-null, especially if the suggestion to switch to a when statement is `SessionControlView` is taken.

Reply:
good point!